installer/templates/phx_single/config/dev.exs

```diff
@@ -18,7 +18,8 @@ config :<%= @app_name %>, <%= @endpoint_module %>,
       "node_modules/webpack/bin/webpack.js",
       "--mode",
       "development",
-      "--watch-stdin",
+      "--watch",
+      "--watch-options-stdin",
```

Comment:
@cw789 one more thing: shouldn't `templates/phx_umbrella/apps/app_name_web/config/dev.exs` also be updated?

Reply:
Probably yes. I wasn't aware of it.
(My first time within the Phoenix repo ;)

Thanks. I'll fix it later.

Reply:
Should I note that then there somewhere as comment?
Or am I allowed to?
I think it's not obvious single install vs. umbrella for newbees.
But possibly I missed something.

Reply:
No worries, we can also update it when we merge. Unfortunately we cannot add comments because they would appear in the user generated file. :)

Reply:
I also missed it :) I missed also another instance: in `installer/templates/phx_assets/package.json` the `watch` command.

Reply:
> `installer/templates/phx_assets/package.json`

True. It's again missing there.
Had it in there earlier - must have overwritten it. 🥴

Reply:
@dvic 

> I also missed it :) I missed also another instance: in `installer/templates/phx_assets/package.json` the `watch` command.

I just checked that again, but for `package.json`: `"watch": "webpack --mode development --watch"` there is no need for `--watch-options-stdin`.
This is nowhere in use and could only be called by `npm run watch` where I don't see any need for `--watch-options-stdin`.

Reply:
Ah you are totally right! It's indeed only necessary when Phoenix is executing Webpack.